HW-0.Rmd

```diff
@@ -9,6 +9,6 @@ Your tasks!
 
 * Change the name of the author above to yours
 * Change the date
-* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](https://www.nhl.com/)
+* Replace the NHL URL with the URL for the document you just posted to Rpubs http://rpubs.com/Christianlehner/hw-0take2
 * Click Knit HTML
```

Comment:
You included the wrong URL here, but whatever. The point HW-0 was to practices submission, not the content.